config/routes.rb

```diff
@@ -4,12 +4,15 @@
   root 'homepages#index'
   
   resources :users
-  resources :products
+  resources :products 
```

Comment:
We now have products nested under users. This may flag a merge conflict, so just keep an eye out for that.